dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/schema/transformers/UserPropertyTransformerTest.java

```diff
@@ -118,13 +136,53 @@ public void testJsonSerializer() throws JsonProcessingException
 
         assertEquals( "usernamea", simple.getUser().getUserCredentials().getUsername() );
         assertEquals( uuid, simple.getUser().getUserCredentials().getUuid() );
+
+        assertNotNull( simple.getUsers() );
+        assertEquals( 4, simple.getUsers().size() );
+
+        /*
+        FieldFilterParams params = new FieldFilterParams(
```

Comment:
commented out code

Reply:
Still working on this one :) was commented out to get it in before the soft freeze (this is for property transfor for collections)